### ISSUE TITLE
Remove /boot RW workaround for kdump

### ISSUE THREAD

**Issue by travier** [coreos/fedora-coreos-tracker#864]
**Describe the bug**

Revert https://github.com/coreos/fedora-coreos-config/pull/936 once we have made sure that all SELinux / kexec-tools issues are resolved.

**Comment by dustymabe** [comment]
Seems like the kexec tools bits landed in rawhide:

- https://src.fedoraproject.org/rpms/kexec-tools/c/d0a301aa3a76e8006dfbfafb5b1e2f6437bd80af?branch=rawhide
- https://lists.fedoraproject.org/archives/list/kexec@lists.fedoraproject.org/thread/73XGH3N7GQS5LCNTUNQQ6NREKLBUCJH6/#73XGH3N7GQS5LCNTUNQQ6NREKLBUCJH6

So we can switch over once we get F35 up and running. We could also speculatively do some of this in the rawhide branch now if we get that passing tests.

Did the SELinux bits land? Is there anything tracking it?

**Comment by travier** [comment]
Ping @kelvinfan001 (if you know the answer, otherwise I'll dig in)

**Comment by kelvinfan001** [comment]
Looks like the SELinux bits have landed in F33 https://bugzilla.redhat.com/show_bug.cgi?id=1951323, so we should be good to drop once we have right kexec-tools version.

**Comment by dustymabe** [comment]
> Looks like the SELinux bits have landed in F33

I assume that implies they've also landed in rawhide/F34.

**Comment by kelvinfan001** [comment]
https://src.fedoraproject.org/rpms/selinux-policy/blob/f34/f/selinux-policy.spec#_858 suggests version `34.8-1` should have what we need. This version is already in our stable stream.

**Comment by dustymabe** [comment]
@travier - do we need to re-open this because of https://github.com/coreos/fedora-coreos-config/pull/1313 ?

**Comment by travier** [comment]
No, this does not seem to be the issue related to the kdump failures.

**Comment by travier** [comment]
I'm keeping https://github.com/coreos/fedora-coreos-tracker/issues/860 open to track new kdump failures

**Comment by dustymabe** [comment]
The fix for this went into stable stream release `34.20211016.3.0`.
